pennylane/ops/__init__.py

```diff
@@ -64,7 +64,7 @@ class Identity(CVObservable):
     eigvals = np.array([1, 1])
 
     def label(self, decimals=None, base_label=None):
-        return super().label(decimals=decimals, base_label=base_label or "I")
+        return base_label or "I"
```

Comment:
How come this was changed?

Reply:
When displaying a circuit where `qml.Identity.inv()` was called, we get a display that looks like `I⁻¹`. For operations like PauliX and Hadamard the inverse is itself, to replicate this for the identity operator I had to make the above change. Those operations re implement the label method in a similar fashion so I think its good.